### PR TITLE
Feature/music-genre-table

### PR DESCRIPTION
**Pull request by Debas-31**
- [x] Create a schema.sql file with tables that will be analogical to the structure of the classes that you created:
   - music_albums table (add all properties and associations from the parent Item class as table columns)
   - genres table

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CREATE DATABASE catalog;
+CREATE TABLE music_albums (
+  id INT,
+  /* name VARCHAR(100),*/
```
Comment (jaflih) [inline database/schema.sql:4]: It is advisable to remove commented code so as to make your code cleaner.

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
+CREATE DATABASE catalog;
+CREATE TABLE music_albums (
+  id INT,
```
Comment (jaflih) [inline database/schema.sql:3]: Each table must have a primary key, so it is better to add it

Comment (jaflih) [inline database/schema.sql:3]: Each table must have a primary key, so it is better to add it

File: database/schema.sql [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  /* name VARCHAR(100),*/
+  on_spotify BOOLEAN,
+  FOREIGN KEY(id) REFERENCES item(id)
```
Comment (jaflih) [inline database/schema.sql:6]: to create inheritance in SQL, you can do:
- like @Ginohmk  did: add all mother properties to the child, so here the table `music_albums` must had: `publish_date`, `archived`, `id`  and `on_spotify`
- like I did: use the `INHERITS` keyword  , there is the code that I used : 
```
CREATE TABLE games(
  multiplayer BOOLEAN,
  last_played_at DATE
) INHERITS (Items);
```

Comment (jaflih) [inline database/schema.sql:6]: to create inheritance in SQL, you can do:
- like @Ginohmk  did: add all mother properties to the child, so here the table `music_albums` must had: `publish_date`, `archived`, `id`  and `on_spotify`
- like I did: use the `INHERITS` keyword  , there is the code that I used : 
```
CREATE TABLE games(
  multiplayer BOOLEAN,
  last_played_at DATE
) INHERITS (Items);
```

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @Debas-31,

Good job !
There are some issues that you still need to work on but you are almost there!

File: database/schema.sql (hunk 3 above)
Comment (Debas-31) [reply database/schema.sql:6]: Thank you for the feedback. I am not sure whether we can use a foreign key as I did instead of the `INHERITS` keyword.

**Review by Ginohmk — APPROVED** [review]
## Approved 

<hr>

You followed all requirements

**Review by jaflih — APPROVED** [review]
Hi,
Congratulations! 🎉
Your project is complete! There is nothing else to say other than... it's time to merge it 🚀 :

Cheers and Happy coding!👏👏👏

**Pull request merged**